buildSrc/src/main/groovy/com/jakewharton/rxbindingproject/KotlinGenTask.groovy

```diff
@@ -1,4 +1,4 @@
-package com.jakewharton.kotlingen
+package com.jakewharton.rxbindingproject
```

Comment:
can we make this `com.jakewharton.rxbinding.project` so it's like a subpackage of the normal project